src/routes/(admin)/questionbank/columns.ts

```
@@ -0,0 +1,51 @@
+	question_text: string;
+	tags?: Array<{ name: string }>;
+	modified_date: string;
+	options: Array<{ id: string; key: string; value: string }>;
+	correct_answer: string[];
+}
+
+export const createQuestionColumns = (
+	currentSortBy: string,
+	currentSortOrder: string,
+	handleSort: (columnId: string) => void,
+	currentPage: number,
```

ESLint

[error] 'currentPage' is defined but never used.

(@typescript-eslint/no-unused-vars)